src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,44 @@
+package duke;
+
+/**
+ * Represents a Personal Assistant Chatbot that helps a person to keep track of various things.
+ */
+public class Duke {
+
+    private final TaskList tasks;
```

Comment:
Preferably have some class members documentation for Duke since Duke is your main program.